dashboard/test/models/census/census_summary_test.rb

```diff
@@ -39,72 +39,77 @@ class Census::CensusSummaryTest < ActiveSupport::TestCase
   end
 
   test "High school with ALL and topic_blocks does teach" do
     submission = build :census_submission, how_many_20_hours: "ALL", topic_blocks: true, topic_text: nil
-    assert Census::CensusSummary.submission_teaches_cs?(submission, is_high_school: true)
+    assert Census::CensusSummary.submission_teaches_cs?(submission, is_high_school: true, is_k8_school: false)
   end
 
   test "High school with ALL and topic_text does teach" do
     submission = build :census_submission, how_many_20_hours: "ALL", topic_blocks: nil, topic_text: true
-    assert Census::CensusSummary.submission_teaches_cs?(submission, is_high_school: true)
+    assert Census::CensusSummary.submission_teaches_cs?(submission, is_high_school: true, is_k8_school: false)
   end
 
   test "High school with ALL and topic_text and topic_blocks does teach" do
     submission = build :census_submission, how_many_20_hours: "ALL", topic_blocks: true, topic_text: true
-    assert Census::CensusSummary.submission_teaches_cs?(submission, is_high_school: true)
+    assert Census::CensusSummary.submission_teaches_cs?(submission, is_high_school: true, is_k8_school: false)
   end
 
   test "Non-high school with NONE and NONE doesn't teach" do
     submission = build :census_submission, how_many_10_hours: "NONE", how_many_20_hours: "NONE"
-    refute Census::CensusSummary.submission_teaches_cs?(submission, is_high_school: false)
+    refute Census::CensusSummary.submission_teaches_cs?(submission, is_high_school: false, is_k8_school: true)
   end
 
   test "Non-high school with 10 hours SOME does teach" do
     submission = build :census_submission, how_many_10_hours: "SOME", how_many_20_hours: "NONE"
-    assert Census::CensusSummary.submission_teaches_cs?(submission, is_high_school: false)
+    assert Census::CensusSummary.submission_teaches_cs?(submission, is_high_school: false, is_k8_school: true)
   end
 
   test "Non-high school with 10 hours ALL does teach" do
     submission = build :census_submission, how_many_10_hours: "ALL", how_many_20_hours: "NONE"
-    assert Census::CensusSummary.submission_teaches_cs?(submission, is_high_school: false)
+    assert Census::CensusSummary.submission_teaches_cs?(submission, is_high_school: false, is_k8_school: true)
   end
 
   test "Non-high school with 20 hours SOME does teach" do
     submission = build :census_submission, how_many_10_hours: "NONE", how_many_20_hours: "SOME"
-    assert Census::CensusSummary.submission_teaches_cs?(submission, is_high_school: false)
+    assert Census::CensusSummary.submission_teaches_cs?(submission, is_high_school: false, is_k8_school: true)
   end
 
   test "Non-high school with 20 hours ALL does teach" do
     submission = build :census_submission, how_many_10_hours: "NONE", how_many_20_hours: "ALL"
-    assert Census::CensusSummary.submission_teaches_cs?(submission, is_high_school: false)
+    assert Census::CensusSummary.submission_teaches_cs?(submission, is_high_school: false, is_k8_school: true)
```

Comment:
Is there a test for `is_k8_school: nil`? According to the comments above that is a valid scenario.

Reply:
Good point. I'll add this.

Reply:
Added.